packages/manager/modules/pci/src/components/project/instance/instance.class.js

```diff
@@ -171,6 +171,10 @@ export default class Instance {
     return includes(get(this, 'image.tags', []), 'application');
   }
 
+  isDeprecatedImage() {
+    return get(this, 'image.status', '') === 'deprecated';
```

Comment:
The third param is not required here ;-) If `image.status` is not found, this willl test `undefined` to a `string` which will return `false`.
Maybe `deprecated` should came from a constants file with the image status enum?

Reply:
So i updated to compare with a value come from an enum.
For the third param, i prefer to use an empty string value to said explicitly i compare a string to string.